kolibri/plugins/coach/assets/src/constants/lessonsConstants.js

```diff
@@ -1,9 +1,7 @@
 export const LessonsPageNames = {
-  PLAN_LESSONS_ROOT: 'PLAN_LESSONS_ROOT',
-  PLAN_LESSONS_ROOT_BETTER: 'PLAN_LESSONS_ROOT_BETTER',
   LESSON_CREATION_ROOT: 'LESSON_CREATION_ROOT',
   LESSON_CREATION_ROOT_BETTER: 'LESSON_CREATION_ROOT_BETTER',
-  SUMMARY: 'SUMMARY',
+  SUMMARY: 'LESSON_SUMMARY',
```

Comment:
If you agree in having these page names in the same `PageNames` object, this will be removed.

Reply:
wfm